tests/src/ondevice_tests/neural_depth_node_test.cpp

```
@@ -9,13 +12,18 @@ extern "C" const char* __asan_default_options() {
+    return std::find(supportedModels.begin(), supportedModels.end(), model) != supportedModels.end();
+}
+
 void testNeuralDepthModelBasic(dai::DeviceModelZoo model, float minFps) {
     constexpr float kFpsTolerance = 0.2f;
 
     dai::Pipeline pipeline;
     auto device = pipeline.getDefaultDevice();
-    if(!device->isNeuralDepthSupported()) {
-        WARN("Skipping NeuralDepth replay test: device doesn't support NeuralDepth.");
+    if(!isModelSupported(device, model)) {
+        WARN("Skipping NeuralDepth replay test: model " << magic_enum::enum_name(model) << " is not supported on this device.");
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)